modules/react/radio/stories/examples/Error.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {FormField} from '@workday/canvas-kit-react/form-field';
+import {Radio, RadioGroup} from '@workday/canvas-kit-react/radio';
+
+export const Error = () => {
+  const [value, setValue] = React.useState<string | number>('thin');
```

Comment:
Nit: I feel like this should be set to `deep-dish` (same as the alert example).

```suggestion
  const [value, setValue] = React.useState<string | number>('deep-dish');
```